apps/src/templates/Lightbulb.jsx

```diff
@@ -123,6 +137,12 @@ export default class Lightbulb extends React.Component {
   }
 }
 
+export default connect(state => {
+  return {
+    isRtl: state.isRtl,
+  };
+})(Radium(Lightbulb));
```

Comment:
Do we need Radium? I don't think we're using any Radium styles if this was working fine before?